scripts/process/find_range.R

```diff
@@ -0,0 +1,18 @@
+#' Extract the min and max to use to scale circles in D3
+#' 
+process.find_range <- function(viz) {
+  
+  deps <- readDepends(viz)
+  wu_data_15_simple <- deps[["wu_data_15_simple"]]
+  col_names <- deps[["col_names"]][["new_names"]]
+  
+  water_use_df <- dplyr::select(wu_data_15_simple, col_names)
+  # wu_data_15_range <- data.frame(
+  #   min = min(water_use_df, na.rm = TRUE),
+  #   max = max(water_use_df, na.rm = TRUE),
+  #   stringsAsFactors = FALSE
+  # )
+  wu_data_15_range <- range(water_use_df, na.rm = TRUE)
+  jsonlite::write_json(wu_data_15_range, viz[["location"]])
```

Comment:
not that we should do this right now, but these range data seem like a good use case for webpack or a templated app.js or something so we don't have to pass a bunch of small files around. i don't know what the performance cost of passing around 1 more tiny file is, but i bet it's bigger than having it embedded in another file to begin with.

Reply:
I agree, I would love to be able to have a better connection between vizlab and js stuff in the future. I know that was one of my investigatory tasks in d3 before we got started on this, but I was never able to actually work on it.

Reply:
yeah, and i think that's fine. just wanted to keep collecting use cases so that when we do find the time and the right approach, we're clear on why we're doing it :+1: